crates/bevy_ecs/src/storage/table.rs

```diff
@@ -539,6 +544,18 @@ impl Column {
             component_ticks.get_mut().check_tick(change_tick);
         }
     }
+    pub fn read_last_mutable_access_tick(&self) -> Tick {
+        Tick::new(
+            self.last_mutable_access_tick
+                .load(std::sync::atomic::Ordering::Relaxed),
+        )
+    }
+    pub fn update_last_mutable_access_tick(&self, tick: Tick) {
+        // self.last_mutable_access_tick
+        //     .store(tick.get(), std::sync::atomic::Ordering::Relaxed);
+        self.last_mutable_access_tick
+            .fetch_max(tick.get(), std::sync::atomic::Ordering::AcqRel);
+    }
```

Comment:
Could you explain the choice for memory ordering? Naively, I would have expected the fetch_max to be `Release` and the load to be `Acquire`, so that we cannot miss changes. However I'm not confident in my understanding of atomic ordering.

Reply:
I haven't deeply considered this aspect yet(so please correct me if i am wrong). My thought was that there might not be any other data flow related to access-tick. Therefore, using a relaxed load operation would be acceptable. Bevy's scheduler may run systems with same table access in parallel, but we cannot guarantee the progress when system would load or store corresponding tick, when two systems write to the same tick concurrently.  store operations must be observed. but for load operation ,even if changes are missed,it is not a big deal,our focus is on comparing the loaded value to the system's last_change_tick.